localisation/00_zuwc_united_localisation.csv

```diff
@@ -1209,12 +1209,11 @@ EVTDESC_WCTHW_2010;The people who have become infected with the plague cannot be
 EVTOPTA_WCTHW_2010;I will save them from their suffering!;;Ich werde sie von ihren Qualen erl�sen!;;;;;;;;;;;x
 EVTOPTB_WCTHW_2010;I'm not a butcher!;;Ich bin kein Schl�chter!;;;;;;;;;;;x
 EVTTITLE_WCTHW_2011;The Culling of [From.GetName];L��puration de [From.GetName];Die S�uberung [From.GetName]s;;;;;;;;;;;x
-EVTDESC_WCTHW_2011_world;Terrible news has reached your court. The Scourge drives people to carry out atrocious acts... [Root.GetTitledFirstName] and [Root.GetHerHis] army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women and children, but [Root.GetFirstName] showed no mercy. After the butchering, [Root.GetSheHe] set the city on fire and left it in ruins.;;Schreckliche Nachrichten haben Euren Hof erreicht. Die Gei�el bringt die Menschen dazu, schreckliche Dinge zu tun.\n[Root.GetTitledFirstName] und [Root.GetHerHis]e Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M�nner, Frauen und Kinder, aber [Root.GetFirstName] zeigte kein Erbarmen. Nach [Root.GetHerHis]er schrecklichen Tat setzte [Root.GetSheHe] die Stadt in Brand und legte sie in Schutt und Asche.;;;;;;;;;;;x
-EVTDESC_WCTHW_2011_arthas;The news of your deeds has reached the courts. The Scourge drives people to carry out atrocious, but necessary, acts... You and your army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women and children, but you showed no mercy. After the butchering, you had set the city on fire and left it in ruins to prevent the dead from rising once again.;;Die Nachricht von Ihren Taten hat die Hof erreicht.
- Die Gei�el bringt die Menschen dazu, schreckliche aber notwendig Dinge zu tun.\nDu und deine Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M�nner, Frauen und Kinder, aber du zeigte kein Erbarmen. Nach dieser schrecklichen Tat setzte du die Stadt in Brand und legte sie in Schutt und Asche um zu verhindern, dass die Toten wieder auferstehen.;;;;;;;;;;;x
+EVTDESC_WCTHW_2011_world;Terrible news has reached your court. The Scourge has driven even the most noble people to carry out atrocious acts... [Root.GetTitledFirstName] and [Root.GetHerHis] army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women, and children, but [Root.GetFirstName] showed them no mercy. Following the butchering, [Root.GetSheHe] had [Root.GetHerHis] men set the city on fire, leaving it in ruins.;;Schreckliche Nachrichten haben Euren Hof erreicht. Die Gei?el bringt die Menschen dazu, schreckliche Dinge zu tun.\n[Root.GetTitledFirstName] und [Root.GetHerHis]e Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M?nner, Frauen und Kinder, aber [Root.GetFirstName] zeigte kein Erbarmen. Nach [Root.GetHerHis]er schrecklichen Tat setzte [Root.GetSheHe] die Stadt in Brand und legte sie in Schutt und Asche.;;;;;;;;;;;x
```

Comment:
Please change german to "Schreckliche Nachrichten haben Euren Hof erreicht. Die Geißel bringt die selbst die edelsten Menschen dazu, grausame Dinge zu verrichten...\n[Root.GetTitledFirstName] und [Root.GetHerHis]e Armee haben alle infizierten Einwohner von [From.GetName] abgeschlachtet. Unter ihnen befanden sich Männer, Frauen und Kinder, aber [Root.GetFirstName] zeigte kein Erbarmen. Nach [Root.GetHerHis]er schrecklichen Tat setzte [Root.GetSheHe] die Stadt in Brand und legte sie in Schutt und Asche."

Reply:
I'll take care of it.